src/pages/Footer/Footer.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Footer.scss';
+
+const Footer = () => {
+  return (
+    <div class="footer">
+      <div class="footerarea1">
+        <Link to="/Product">로그인</Link>
+        <Link to="/Product">고객센터</Link>
+        <Link to="/Product">브랜드 스토리</Link>
+        <Link to="/Product">블로그</Link>
+        <Link to="/Product">인재채용</Link>
```

Comment:
현재 안만들어진 페이지에대해서는 #을 붙히는게 관습이긴 합니다.
이렇게 바꾸는게 다른 개발자가 읽기에 "아~ 아직 페이지가 안만들어져서 to를 붙히지 못했구나" 를 생각할 수 있을 것 같습니다.

```suggestion
        <Link to="#">로그인</Link>
        <Link to="#">고객센터</Link>
        <Link to="#">브랜드 스토리</Link>
        <Link to="#">블로그</Link>
        <Link to="#">인재채용</Link>
```